voorhoede/heads-up
# Fix/light & dark theme

## What change(s) were made
- Dark theme now has the correct dark background color.

## How to test
- Run the Chrome extension locally and check if the extension has the correct color theme when switching between light/dark.

## Related Trello ticket(s)
- [https://trello.com/c/te5d7mx5](https://trello.com/c/te5d7mx5)

## Checks
